metron-interface/metron-alerts/scripts/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "metron-management-ui-web-server",
```

Comment:
Was this copied from metron-config?  I think this file needs to be updated to match this module.